docs/development/local-development.md

```diff
@@ -75,10 +75,16 @@ To build the correct docker image:
 docker build -f .devcontainer/Dockerfile -t renovatebot_local .
 ```
 
+Starting from Docker Engine 23.0 and Docker Desktop 4.19, Docker uses Buildx by default.
+So you must run the following command to get the image loaded to the Docker image store:
+```
```

Comment:
I don't know why GitHub strips my newlines. You'll have to put the newline in yourself. 😉